src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -406,8 +406,8 @@ private void assertAddressIsTransformed(Map<String, Object> address, Map<String,
     private void assertChildrenInfo(Map<String, Object> transformedCaseData) {
         List<Map<String, Object>> children = (List)(transformedCaseData.get("childrenInfo"));
 
-        assertChild(children.get(0), asList("Johny Bravo", "2000-03-12", "male", "son", "SON","New Zeeland"));
-        assertChild(children.get(1), asList("Anne Shirley", "1895-03-12", "female", "daughter", "Daughter","Canada"));
+        assertChild((Map)(children.get(0)).get("value"), asList("Johny Bravo", "2000-03-12", "male", "son", "SON","New Zeeland"));
+        assertChild((Map)(children.get(1)).get("value"), asList("Anne Shirley", "1895-03-12", "female", "daughter", "Daughter","Canada"));
     }
 
     private void assertChild(Map<String, Object> child, List<String> values) {
```

Comment:
I think I'd have the second parameter here be a map. Either that, or just compare using "hasEntry()" in the test itself.

Reply:
It's being refactored :)

#stayTunned (at home ;) )